Fixed start timeout on ubuntu 16.04 and debian 8. Fixed kitchen tests on docker.

rremmel opened the pull request:
### Description

Fix start timeout on ubuntu 16.04. 
Change systemd Service Type=notify to Type=simple on Ubuntu. Apache on ubuntu 16.04 does not come with systemd integration, so the type=notify can't be used. 
### Issues Resolved

[List any existing issues this PR resolves]
### Check List
- [x] All tests pass. See https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] The CLA has been signed. See https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD

Apparently apache on Ubuntu 16.04 does not contain systemd support. This means that systemd Service type=notify can't be used.
This change adds a condition to the template so that Type=simple would be used on ubuntu.

Commit:
fixed systemd start timeout on ubuntu 16.04

Apparently apache on Ubuntu 16.04 does not contain systemd support. This means that systemd Service type=notify can't be used.
This change adds a condition to the template so that Type=simple would be used on ubuntu.

chef-supermarket commented:
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

Commit:
fix startup timeout on debian-8.

Commit:
Fix integration tests on opensuse.

Apache modules in Opensuse are stored in /usr/lib64/apache2 not in /usr/lib64/httpd/modules as on rhel.

Commit:
Fixed kitchen integration tests on docker.

Tests that verified service ports failed on multiple platforms because netstat utility used by inspec was not installed.

chef-supermarket commented:
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

Commit:
fixed cookstyle test failure

Offenses:
test/integration/module24/rhel_spec.rb:66:1: C: 3 trailing blank lines detected.
63 files inspected, 1 offense detected

iennae commented:
This looks good. The failures appear to be unrelated to what you have got so I'm going to go ahead and pull it in.
